pkg/plugin/plugin.go

```diff
@@ -84,10 +93,23 @@ func (j *JiraPlugin) Validate(ctx context.Context, req *jvspb.ValidateJustificat
 		}, fmt.Errorf("failed to validate justification: %w", err)
 	}
 
+	// There is only one JQL and one issueKey, so the first match result is
+	// checked directly.
+	if len(result.Matches[0].MatchedIssues) == 1 {
+		issueID := strconv.Itoa(result.Matches[0].MatchedIssues[0])
+		issueURL := j.baseURL + req.Justification.Value
```

Comment:
We would need to add the slash here: `j.baseURL + "/" + req.Justification.Value`

Reply:
Update the issueURL as url.JoinPath(j.issueBaseURL, "/browse/", req.Justification.Value)